tests/nested/manual/grade-signed-cloud-init-testkeys/task.yaml

```diff
@@ -146,9 +146,8 @@ execute: |
     fi
     sleep 1
   done
-
+  
   echo "Ensuring that cloud-init got disabled after running"
-  nested_exec "cloud-init status" | MATCH "status: disabled"
```

Comment:
same please leave this line in

Reply:
@sergiocazzolato this line still seems to be deleted unless the diff is wrong somehow?

Reply:
done